apps/condo/domains/organization/utils/testSchema/index.js

```diff
@@ -50,7 +51,22 @@ async function createTestOrganization (client, extraAttrs = {}) {
     const obj = await Organization.create(client, attrs)
     return [obj, attrs]
 }
-
+async function createDefaultOrganizationEmployeeRoles(client, organization, extraAttrs = {}) {
```

Comment:
It’s better not to add a new test until which will be used only once. 
you can use makeClientWithOrganization